ui/helpers/job.js

```diff
@@ -237,6 +237,15 @@ export const addAggregateFields = function addAggregateFields(job) {
   return job;
 };
 
+// Remove any fields we don't need for display in the greater job field after
+// they have been mapped/processed.
+export const trim = function trim(job) {
```

Comment:
So these are used "under the hood" in the UI to create variables but then deleted for user display purposes?

Reply:
Yeah, exactly.  :)